### PR TITLE
docs: add VG-Lite related documents

### PR DESCRIPTION
**Pull request by FASTSHIFT**
### Description of the feature or fix

Added VG-Lite rendering backend and VG-Lite simulator usage related documents.

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

File: docs/overview/vg_lite_tvg.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+=================
+VG-Lite Simulator
+=================
+
+LVGL integrates a VG-Lite simulator based on ThorVG.
+Its purpose is to simplify the debugging of VG-Lite adaptation and reduce the time of debugging and locating problems on hardware devices.
+
+It has been integrated into the CI automated compilation and testing process to ensure that the VG-Lite rendering backend can be fully tested after each PR modification.
+
+You can find the standalone version `here <https://github.com/FASTSHIFT/vg_lite_tvg/>`_.
```
Comment (kisvegabor) [inline docs/overview/vg_lite_tvg.rst]: We shouldn't link to to user repos. If needed we can add content similar to your README here.

**Review by kisvegabor — COMMENTED** [review]
Wonderful! Thank you! :heart:

File: docs/overview/vg_lite_tvg.rst (hunk 1 above)
Comment (FASTSHIFT) [reply docs/overview/vg_lite_tvg.rst]: Done, I remove it.

**Review by kisvegabor — APPROVED** [review]

**Review by liamHowatt — APPROVED** [review]
Well written!

**Pull request merged**